### ISSUE TITLE
BND warning 

### ISSUE THREAD

**Issue by joerghoh** [Adobe-Consulting-Services/acs-aem-commons#1813]
The build throws this warning

```
[INFO] --- maven-bundle-plugin:3.4.0:manifest (generate-scr-metadata-for-unittests) @ acs-aem-commons-bundle ---
[WARNING] Manifest com.adobe.acs:acs-aem-commons-bundle:bundle:4.0.1-SNAPSHOT : Export com.adobe.acs.commons.mcp.model,  has 1,  private references [com.adobe.acs.commons.mcp.model.impl]
```

I found that ``com.adobe.acs.commons.mcp.model.ManagedProcess`` (which is an exported class) uses the class ``com.adobe.acs.commons.mcp.model.impl.ArchivedProcessFailure`` (a non-exported class) in publich method signatures. For me it seems that this method can not be used by anyone outside of the acs-aem-commons bundle.

@badvision as you wrote most of the code here, have you tried that already?

**Comment by badvision** [comment]
I've seen the warning just haven't gotten around to resolving that one.

**Comment by kwin** [comment]
@joerghoh Was this fixed with https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/1814?

**Comment by badvision** [comment]
@joerghoh Wasn't this fixed in a recent PR from you?

**Comment by davidjgonzalez** [comment]
@joerghoh was this fixed? Should we close as part of 4.2.0?
